### PR TITLE
feature: add is_mining for ePIC, add T21, add BlockMiner 720i

### PR DESCRIPTION
**Pull request by jpcomps**

### CODE REVIEW

File: pyasic/miners/backends/epic.py [hunk 1]
```diff
@@ -313,8 +317,22 @@ async def _get_hashboards(
...
+        if web_summary is not None:
+            try:
+                op_state = web_summary["Status"]["Operating State"]
+            except KeyError:
+                return None
+                pass
+        if op_state != "Idling":
+            return True
+        else:
+            return False
```
Comment (b-rowan) [inline pyasic/miners/backends/epic.py:335]: I would just `return not op_state == "idling"` in your try block, then you can just pass on the key error and not have to do more parsing.

**Review by b-rowan — CHANGES_REQUESTED** [review]

File: pyasic/miners/backends/epic.py (hunk 1 above)
Comment (jpcomps) [reply pyasic/miners/backends/epic.py]: done. much cleaner. thanks

**Review by b-rowan — APPROVED** [review]

**Comment by b-rowan** [comment]
Yet another Antminer supported on ePIC before stock firmware is supprorted haha.

Ill do up stock support really quick, then push a version with the changes, thanks!

**Comment by jpcomps** [comment]
i really should just push the stock support too, thats my bad. Next time I promise :)

**Pull request merged**